cmd/operator/start.go

```diff
@@ -76,9 +83,9 @@ func main() {
 	}
 	// +kubebuilder:scaffold:builder
 
-	setupLog.Info("starting manager")
+	setupLog.Info("starting contour-operator")
 	if err := mgr.Start(ctrl.SetupSignalHandler()); err != nil {
-		setupLog.Error(err, "problem running manager")
+		setupLog.Error(err, "problem running contour-operator")
```

Comment:
nit: "failed to start contour-operator" would be a marginally better error

Reply:
ack